dbt_project.yml

```diff
@@ -13,6 +14,13 @@ snapshot-paths: ["snapshots"]
 vars:
     linkedin__use_local_currency: true
```

Comment:
can you embed this variable under `linkedin_source`? Similar to the other source variables